ajax/libs/bootstrap3-wysihtml5-bower/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "bootstrap3-wysihtml5-bower",
+  "version": "0.3.3",
+  "filename": "bootstrap3-wysihtml5.all.min.js",
+  "description": "Bootstrap 3 compatible wysiwyg editor",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/Waxolunist/bootstrap3-wysihtml5-bower.git"
+  },
+  "keywords": [
+    "bootstrap",
+    "wysiwyg"
+  ],
+  "author": "Christian Sterzl <christian.sterzl@gmail.com>",
+  "license": "MIT",
+  "homepage": "https://github.com/Waxolunist/bootstrap3-wysihtml5-bower",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/bootstrap-wysiwyg/bootstrap3-wysiwyg.git",
+    "basePath": "dist",
+    "files": [
+      "**/*"
```

Comment:
Because we don't host `json` files in "dist/parser_rules/" folder, `"**/*.+(js|css)"` would be better.